packages/cf-layout/src/organisms/featured-content-module.less

```diff
@@ -19,25 +19,31 @@
         padding-top: unit( @grid_gutter-width / @base-font-size-px, em );
         padding-bottom: unit( @grid_gutter-width / @base-font-size-px, em );
 
+        .respond-to-max( @bp-xs-max, {
+            padding-right: unit( ( @grid_gutter-width / 2 ) / @base-font-size-px, em );
```

Comment:
TRIVIAL: This and the line below have extra parens you don't need. `unit( @grid_gutter-width / 2 / @base-font-size-px, em );` should work.